docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,49 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Examples
+
+### Download Files in IE
+
+TestCafe prevents native dialogs before file download in all browsers except Internet Explorer. This dialog prevents automatic file download but does not block the page.
+
+The following example shows how to download the file despite the dialog:
+
+```html
+<body>
+    <form method="get" action="./src/file.zip">
+        <button id="downloadButton" type="submit">Download!</button>
+     </form>
+</body>
+```
+
+This sample page includes a button that downloads a file from the `/src` folder on the server. To obtain the file, use a [RequestLogger](../../reference/test-api/requestlogger/README.md):
+
+```js
+import { RequestLogger } from 'testcafe';
+
+const fileDownloadLogger = RequestLogger(new RegExp('src/file.zip'), {
+    logResponseBody: true,
+    stringifyResponseBody: true
+});
+
+fixture `fixture`
+    .page `./fileDownload.html`
+    .requestHooks(fileDownloadLogger);
+
+test(`Download a file and verify contents`, async t => {
+
+    await t
+        .click('#downloadButton')
+
+    await t
+        .expect(fileDownloadLogger.contains(r => {
+            return  /File contents here/.test(r.response.body) &&   //verify response body
+                    r.response.statusCode === 200;                  //verify response status code
+        }))
+        .ok()
+});
+```
+
+This test introduces a `RequestLogger` that logs requests to a location and received responses. Location is [defined with a regular expression](../../../templates/intercept-http-requests/request-filter.md#use-a-regular-expression-to-specify-the-url). Stringified response body is then evaluated with a regular expression.
```

Comment:
Can you confirm that "stringified" is a widely used technical term?

```suggestion
This test introduces a `RequestLogger` that logs requests to a location and the responses received. The location is [defined with a regular expression](../../../templates/intercept-http-requests/request-filter.md#use-a-regular-expression-to-specify-the-url). The stringified response body is then evaluated with a regular expression.
```

Reply:
The word is widely used. It is just the name of a method that converts data to a string. What I meant to say is:
we receive data, then convert it to a string, then evaluate.

> The response body is then evaluated with a regular expression.
>
> Response body received from the server is binary. Use RequestLogger's [stringifyResponseBody option](link here) to convert it to a string.

Reply:
Let's change this to:

`The response body is then evaluated with a regular expression.`

`The response body received from the server is binary. Use the RequestLogger's [stringifyResponseBody option](link here) to convert it to a string.`

Reply:
Done